tests/actions.go

```diff
@@ -2303,12 +2303,7 @@ func (oa *operatorActions) StartValidatingAdmissionWebhookServerOrDie(info *Oper
 	if err != nil {
 		err = fmt.Errorf("failed to start webhook server %v", err)
 		glog.Error(err)
-		sendErr := slack.SendErrMsg(err.Error())
-		if sendErr != nil {
-			glog.Error(sendErr)
-		}
-		// TODO use context instead
-		os.Exit(4)
+		slack.NotifyAndPanic(err)
```

Comment:
`slack.NotifyAndPanic ` can't exit the process, because this is recovered by http server: https://github.com/golang/go/blob/master/src/net/http/server.go#L1764

we should use context instead. we can use os.Exit() now.

Reply:
This panic out of http server's handler func, so this panic cannot recover by the http server